src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -914,12 +916,12 @@ def load_state(self, model_id: str, evaluation_time: float, epsilon: float = 1e-
             ss_filepath = ss_registry.return_saved_state_filepath(ss)
 
             # Initialize model accordingly and load existing state
-            self._initialize_evaluation(parameters=ss.parameters, inputs=ss.inputs)
+            self._initialize_evaluation(parameters=ss.parameters, inputs=ss.inputs, runtime_init=False)
```

Comment:
@lboucin I suppose there was an issue here before when the runtime was already initialized and we re-did initialization a  second time, which explains the need of having that new argument runtime_init, is that correct ?